ome/omero-install
# Redis for osx

This PR test redis instalation on OSX, see doc https://github.com/openmicroscopy/ome-documentation/pull/1431

To test:
- Install web 
- Install and configure redis as stated in https://github.com/aleksandra-tarkowska/omero-install/blob/dev_redir_osx/osx/step01_deps_experimental.sh#L26-34
- monitor redis 
  
  ```
  redis-cli
  127.0.0.1:6379> PING
  PONG
  127.0.0.1:6379> MONITOR
  OK
  (here you will see the content)
  ```
- log in to web, redis monitor should show:
  
  ```
  1460330344.663488 [0 127.0.0.1:55185] "SELECT" "1"
  1460330344.663733 [1 127.0.0.1:55185] "EXISTS" ":1:django.contrib.sessions.cachehd4570rlkho9e6ngpy3qr8cvakpzpk1b"
  1460330344.664056 [1 127.0.0.1:55185] "SETNX" ":1:django.contrib.sessions.cachehd4570rlkho9e6ngpy3qr8cvakpzpk1b" "\x80\x02}q\x01U\tconnectorq\x02comeroweb.connector\nConnector\nq\x03)\x81q\x04}q\x05(U\tis_secureq\x06\x89U\tserver_idq\aX\x01\x00\x00\x001U\auser_idq\b\x8a\x00U\x11omero_session_keyq\tU$162d57e5-34db-47b1-93b7-4262258d2b23q\nU\tis_publicq\x0b\x89ubs."
  1460330344.664377 [1 127.0.0.1:55185] "EXPIRE" ":1:django.contrib.sessions.cachehd4570rlkho9e6ngpy3qr8cvakpzpk1b" "86400"
  1460330344.681574 [1 127.0.0.1:55164] "GET" ":1:django.contrib.sessions.cachehd4570rlkho9e6ngpy3qr8cvakpzpk1b"
  1460330344.760156 [1 127.0.0.1:55164] "DEL" ":1:django.contrib.sessions.cachehd4570rlkho9e6ngpy3qr8cvakpzpk1b"
  ```

https://github.com/ome/omero-install/pull/101/commits/862b15d72daeb3584518fd6f31c43c2bbea1162e is tested via travis
